src/main/java/com/Leapwork/Leapwork_plugin/PluginHandler.java

```diff
@@ -442,10 +444,16 @@ public boolean stopRun(String controllerApiHttpAddress, UUID runId, String sched
                     listener.error(String.format(Messages.STOP_RUN_FAIL,scheduleTitle,runId.toString()));
 
             }
-        } catch (Exception e)
+        } 
+        
+        catch (RuntimeException e) {
```

Comment:
what is the reason to catch `RuntimeException` to throw it again wrapped into another `RuntimeException`?

Reply:
removing it